docker-compose/ckan-conf-templates/datagov-theme-production.ini.template

```diff
@@ -83,11 +83,11 @@ ckan.plugins = stats text_view image_view recline_view datastore xloader
     structured_data geodatagov geodatagov_miscs z3950_harvester 
     arcgis_harvester geodatagov_geoportal_harvester waf_harvester_collection 
     geodatagov_csw_harvester geodatagov_doc_harvester geodatagov_waf_harvester spatial_query 
-    datajson_harvest datajson
+    usmetadata datagovtheme datajson_harvest datajson spatial_metadata
```

Comment:
We do not need to activate the _usmetadata_ plugin here. It's just for the _inventory_ site. Here we work with the catalog. Am I ok @akariv?

Reply:
Yes, that's true. 
Also note that while this file is used in development, it's not part of the production setup
(This is the one: https://github.com/ViderumGlobal/ckan-cloud-helm/blob/master/ckan/templates/ckan-configmap.yaml#L38)